docker-for-windows/release-notes.md

```diff
@@ -13,6 +13,22 @@ This page contains information about the new features, improvements, known issue
 For information about Edge releases, see the [Edge release notes](edge-release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.2.0.4
+2020-03-
+
+### Bug fixes and minor changes
+
+- Docker Desktop now displays hidden files in shared volumes. Fixes [docker/for-win#5808](https://github.com/docker/for-win/issues/5808).
+- Docker Desktop now generates the inotify `MODIFY` events on shared filesystems for Windows file sharing. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530).
+- Attempts to create files in a shared volume with an identical filename but a different case (uppercase/lowercase) on Windows filesystem will now fail with the error `EEXIST`. Fixes [docker/for-win#5894](https://github.com/docker/for-win/issues/5894).
+- Fixed cache invalidation and event injection in shared volumes with host paths longer than 260 characters.
+- Docker Desktop now allows users to rename open files in shared volumes. Fixes [docker/for-win#5565](https://github.com/docker/for-win/issues/5565).
+- Fixed an issue where containers did not synchronize time with the host when the host machine is put to sleep. Fixes [docker/for-win#4526](https://github.com/docker/for-win/issues/4526).
+- Fixed an issue which caused Docker Desktop Dashboard to attempt connecting to all exposed ports inside a container. Fixes [docker/for-win#5903](https://github.com/docker/for-win/issues/5903) and [docker/for-mac#4264](https://github.com/docker/for-mac/issues/4264).
```

Comment:
nit: maybe we should put the link to for-win here and the one to for-mac in the mac section?

Reply:
That's a good idea. I've updated accordingly.